### PR TITLE
#166790010 Users Should Be Able to See Time It Takes To Read An Article

### PR DESCRIPTION
**Pull request by elemanhillary-zz**
#### What does this PR do?
- Adds article read time in human readable string

#### Description of Task to be completed
- Enables users to view article read time

#### How should this be manually tested?
```shell
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-article-read-time-166790010
npm install
npm start
```
- Navigate to `localhost:3000/api/articles/:slug`
- check the for `time` field in the response body

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
[166790010](https://www.pivotaltracker.com/story/show/166790010)
#### Screenshots (if appropriate)
Imgur
#### Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — DISMISSED** [review]
ready to be merged

**Review by IsaiahRn — DISMISSED** [review]
It can be merged!

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — DISMISSED** [review]
It can be merged

**Review by IsaiahRn — DISMISSED** [review]
ready to be merged!

**Review by kodek-sleuth — DISMISSED** [review]

**Review by niyoceles — APPROVED** [review]

**Review by IsaiahRn — DISMISSED** [review]
ready to merge

File: swaggerSetUp/ah-92explorers-api.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const swaggerDefinition = {
     description:
       'Create a community of like minded authors to foster inspiration and innovation by leveraging the modern web',
   },
-  host: 'ah-92explorers-api.herokuapp.com',
+  host: 'localhost:3000',
```
Comment (Hervera) [inline swaggerSetUp/ah-92explorers-api.js]: Reverse to Heroku before you push

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — DISMISSED** [review]

**Review by kodek-sleuth — DISMISSED** [review]

**Review by IsaiahRn — DISMISSED** [review]

**Review by IsaiahRn — DISMISSED** [review]

**Review by niyoceles — DISMISSED** [review]

**Review by Hervera — COMMENTED** [review]
Ready to be merged

**Review by Hervera — DISMISSED** [review]
Ready to be merged

**Review by niyoceles — APPROVED** [review]

**Review by Hervera — APPROVED** [review]
Ready to be merged

**Review by IsaiahRn — APPROVED** [review]

**Pull request merged**